src/style.css

```diff
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Don't forget to add a fallback font in case the main one doesn't work. It's a CSS best practice to have a fallback font that is a system font and widely available across all systems. For example: `font-family: Roboto, Arial, sans-serif;`